test/Libraries/DynamoMSOfficeTests/ExcelTests.cs

```diff
@@ -1,23 +1,25 @@
-using System;
+using DSOffice;
+using NUnit.Framework;
 using System.Collections.Generic;
-using System.Diagnostics;
 using System.IO;
 using System.Linq;
-using CoreNodeModels.Input;
-using DSOffice;
-using Dynamo.Configuration;
-using Dynamo.Graph.Connectors;
-using Dynamo.Graph.Nodes;
-using NUnit.Framework;
-using ProtoCore.DSASM;
-using ProtoCore.Mirror;
+using System.Threading;
 
 
 namespace Dynamo.Tests
 {
-    [TestFixture, RequiresSTA]
+
+#if NET6_0_OR_GREATER
+    [TestFixture, Apartment(ApartmentState.MTA)]
+    public class ExcelTests
+    {
+    }
+#elif NETFRAMEWORK
```

Comment:
So effectively, you're excluding all tests from running in .net6?

Reply:
Yes, I'm excluding all test because ExcelTests is inheriting from DynamoViewModelUnitTest class which is not yet present in Dynamo.All.Net6.sln

Reply:
Yes, I understand that. I was only trying to understand, then why start this PR in the first place until we first migrate the WPF projects or figure out a way to first decouple Excel tests from anything that had to do with WPF like inheriting from a model test class instead of `DynamoViewModelUnitTest`.